browser/model/jbds-autoinstall.js

```diff
@@ -12,7 +12,7 @@ class JbdsAutoInstallGenerator {
   }
 
   generate(jbdsInstallDir, jdkInstallDir) {
-    let jbdsVersion = "9.1";
+    let pjson = require('../../package.json');

```

Comment:
OK, so I can switch to the other file and parse it out of the jbds.jar URL ... but that doesn't help with the other problem re: "no such file or directory" ... does it?